packages/rest/src/lib/utils/constants.ts

```diff
@@ -34,7 +48,12 @@ export const DefaultRestOptions = {
 	handlerSweepInterval: 3_600_000, // 1 Hour
 	async makeRequest(...args) {
 		const strategy = await getUndiciRequest();
-		return strategy.makeRequest(...args);
+
+		if (strategy === null) {
+			return fetch(...args);
```

Comment:
This won't work in node 16. It's probably not possible to reach, but stupider stuff has happened.